nlptest/nlptest.py

```diff
@@ -130,7 +130,7 @@ def generate(self) -> "Harness":
             raise RuntimeError("Please call .configure() first.")
 
         tests = self._config['tests']
-        self._testcases = TestFactory.transform(self.data, tests, self.model)
+        self._testcases = TestFactory.transform(self.data, tests)
```

Comment:
We're not using the model anymore for transform? how do we get expected results for robustness in that case?

Reply:
I will deal with issues in a different way